[FATAL] Testing exception
==CODE==
import std.io.*;
throw "ae";
==RESULT==
Fatal error: Unhandled exception! on line 1
Message: ae
Stack trace:

